src/main/java/io/axoniq/axonserver/connector/admin/LoadBalanceStrategy.java

```diff
@@ -0,0 +1,37 @@
+package io.axoniq.axonserver.connector.admin;
+
+/**
+ * @author Stefan Dragisic
+ */
+public enum LoadBalanceStrategy {
+
+    DEFAULT("default"),
+    THREAD_NUMBER("threadNumber");
+
+    private final String name;
+
+    LoadBalanceStrategy(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public io.axoniq.axonserver.grpc.admin.LoadBalanceStrategy toGrpc() {
+        switch (this) {
+            case DEFAULT : return io.axoniq.axonserver.grpc.admin.LoadBalanceStrategy.DEFAULT;
```

Comment:
I'd wager @MGathier his point on not using an enumeration in the axonserver-api will have some impact on this method.